src/content/about/metrics.md

```diff
@@ -0,0 +1,85 @@
+| Script Threw Exception | Worker script threw an unhandled Javascript exception                    | 1101               | `scriptThrewException` |
+| ¹ Exceeded Resources   | Worker script exceeded runtime limits                                    | 1102, 1027         | `exceededResources`    |
+| ² Internal Error       | Workers runtime encountered an error                                     |                    | `internalError`        |
+
+_¹ The Exceeded Resources status may appear when the Worker exceeds a [runtime limit](/about/limits/#request-limits). The most common cause is excessive CPU time, but is also caused by a script exceeding startup time or free tier limits._
+
+_² The Internal Error status may appear when the Workers runtime fails to process a request due to an internal failure in our system. These errors are not caused by any issue with the Worker code nor any resource limit. While requests with Internal Error status are rare, we expect that some may appear during normal operation. These requests are not counted towards usage for billing purposes. If you notice an elevated rate of requests with Internal Error status, please check www.cloudflarestatus.com._
+
+### Requests
+
+This chart shows historical request counts from the Workers runtime broken down into successful requests, errored requests, and subrequests.
+
+- **Total**: All incoming requests registered by a Worker script. Requests blocked by [WAF](https://www.cloudflare.com/waf/) or other security features will not count.
+- **Success**: Requests that returned a Success or Client Disconnected invocation status.
+- **Errors**: Requests that returned a Script Threw Exception, Exceeded Resources, or Internal Error invocation status.
+- **Subrequests**: Requests triggered by calling `fetch` from within a Worker script. A subrequest that throws an uncaught error will not be counted.
+
+Request traffic data may display a "drop off" near the last few minutes displayed in the graph for time ranges less than six hours. This does not reflect a drop in traffic, but a slight delay in aggregation and metrics delivery.
+
+### CPU Time
+
+The CPU time chart shows historical CPU time data broken down into relevant quantiles using [reservoir sampling](https://en.wikipedia.org/wiki/Reservoir_sampling). You can learn more about interpreting quantiles [here](https://www.statisticshowto.com/quantile-definition-find-easy-steps/). In some cases, higher quantiles may appear to exceed [CPU time limits](/about/limits/#cpu-execution-time-limit) without generating invocation errors because of a mechanism in the Workers runtime that allows rollover CPU time for requests below the CPU limit.
+
+## Zone Metrics
+
+Aggregates request data for all scripts assigned to any [routes](/about/routes/) defined for a zone. On your Cloudflare dashboard, click the analytics tab to [view the dashboard](https://dash.cloudflare.com/?zone=analytics/workers).
```

Comment:
Recommending this copy edit here since we're able to link directly to the tab we don't have to advise where to click. 

`Aggregates request data for all scripts assigned to any [routes](/about/routes/) defined for a zone. You’ll find Zone Metrics on the [analytics tab](https://dash.cloudflare.com/?zone=analytics/workers) of your Cloudflare dashboard.`

Reply:
👍